### PR TITLE
Support VS Code and VS for Mac in Mongo tutorial

### PR DESCRIPTION
**Pull request by scottaddie**
Fixes https://github.com/aspnet/Docs/issues/9698

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mongo-app?view=aspnetcore-2.2&branch=pr-en-us-9737&tabs=visual-studio)

### CODE REVIEW

File: aspnetcore/tutorials/first-mongo-app.md [hunk 1]
```diff
@@ -25,11 +25,28 @@ In this tutorial, you learn how to:
...
 ## Prerequisites
 
-* [.NET Core SDK 2.1 or later](https://www.microsoft.com/net/download/all)
-* [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)
-* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.7.3 or later with the following workloads:
+# [Visual Studio](#tab/visual-studio)
+
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.9 or later with the following workloads:
   * **.NET Core cross-platform development**
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mongo-app.md]: x-plat is automatic now - right?

Comment (scottaddie) [reply aspnetcore/tutorials/first-mongo-app.md]: What do you mean by "automatic"?

Comment (Rick-Anderson) [reply aspnetcore/tutorials/first-mongo-app.md]: Automatically added. You don't need to add x-plat. Adding asp.net adds xplat.

Comment (scottaddie) [reply aspnetcore/tutorials/first-mongo-app.md]: In this case, everything works without the x-plat workload. I'll submit another PR to address it.

**Pull request merged**